packages/mwp-app-render/src/util/seo/links.js

```diff
@@ -28,14 +28,16 @@ export const generateCanonicalUrl = (
  * @param  {String} baseUrl base url of the page (protocol + hostname)
  * @param  {String} localeCode   locale of user
  * @param  {String} forcedLocaleCode  locale of group
+ * @param {String} groupCountry The group's country
  * @param  {String} route   redux'd route to the current page
  * @return {Array}  array of React.element's
  */
 export const generateCanonicalUrlLinkTags = (
 	baseUrl,
 	localeCode,
 	route,
-	forcedLocaleCode = ''
+	forcedLocaleCode = '',
+	groupCountry
```

Comment:
web-platform shouldn't be aware of groups or any data from consumer Apps, Let's rename this to smth like `isGenerateAlternateLinks`

Reply:
fixed, updated groupCountry to isGenerateAlternateLinks